backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Controllers/UserController.java

```diff
@@ -29,5 +33,17 @@ public ResponseEntity<User> saveUser(@RequestBody User user){
     User newUser = userService.saveUser(user);
     return ResponseEntity.status(HttpStatus.CREATED).body(newUser);
    }
+
+   @PutMapping("/{id}")
+    public ResponseEntity<String> updateUser(
+            @PathVariable("id") Long id, 
+            @RequestBody User user) {
+        boolean updated = userService.updateUser(id, user);
+        if (updated) {
+            return ResponseEntity.ok("Usuario actualizado exitosamente");
+        } else {
+            return ResponseEntity.status(404).body("Usuario no encontrado");
+        }
```

Comment:
in english plis:
`return ResponseEntity.ok("Usuario actualizado exitosamente");
        } else {
            return ResponseEntity.status(404).body("Usuario no encontrado");`